src/telliot_feeds/queries/query_catalog.py

```diff
@@ -150,3 +151,4 @@
 query_catalog.add_entry(
     tag="string-query-example", title="String query example", q=StringQuery(text="Where is the Atlantic ocean?")
 )
+query_catalog.add_entry(tag="tellor-rng-example", title="Tellor RNG", q=TellorRNG(timestamp=1660567612))
```

Comment:
Is this a valid timestamp? Did you check using the build feed flag and reporting on chain? Maybe link the example tx in the convo so I can decode the submit val bytes. Or you could and just copy/paste the output from the CLI

Reply:
By valid timestamp, I mean will the reporter be able to fetch block hashes form its underlying APIs using that timestamp?

Reply:
ok, thanks for the explanation. here is a transaction for TellorRNG query type using that timestamp.
https://mumbai.polygonscan.com/tx/0x5e1c397538cf2bc97da89409b15adcf100518fdb6b172970fbb5d5fe1aba0c7d

Reply:
Np. Awesome, ty!